phpdotnet/phd/Format.php

```diff
@@ -395,6 +395,10 @@ final public static function autogen($text, $lang = null) {
 
         $filename = Config::lang_dir() . $lang . ".ini";
 
+        if (!file_exists($filename) && strncmp(basename($filename), 'doc-', 4) === 0) {
```

Comment:
```suggestion
        if (!file_exists($filename) && strncmp(basename($filename), 'doc-', 4) === 0) {
```
Why not use ``str_starts_with()`` ?

Reply:
Yeah it used to be that we cannot use PHP 8 because the doc server was running 7.3, but this is not the case anymore and I used it in https://github.com/php/phd/commit/365cd394f1b32a4b9583043ca10f8a2a9d07ba19#diff-c372d6edb12a842ef8a20b14089597d81f1b11f5e0fb7f98f9f305528b133ef5R892 :)

Probably a cleanup to do then :)